applications/asset_tracker_v2/src/cloud/cloud_codec/nrf_cloud/nrf_cloud_codec.c

```diff
@@ -464,6 +464,12 @@ int cloud_codec_decode_config(char *input, size_t input_len,
 		return -ENOENT;
 	}
 
+	/* Check for an nRF Cloud message */
+	if ((json_object_decode(root_obj, DATA_GROUP) != NULL) ||
```

Comment:
Wouldn't this check prevent decoding of incoming shadow messages (config updates)?

Reply:
this checks for "messageType" or "appId". 
shadow updates are not formatted that way, unless i'm missing something? 
the remainder of `cloud_codec_decode_config` looks for "config" and "state" to handle config updates.

Reply:
No you're not, I misunderstood the check. That's what you get for reviewing PRs to early in the morning 😄